src/zenml/cli/stack.py

```diff
@@ -1143,16 +1143,28 @@ def _import_stack_component(
 @stack.command("import", help="Import a stack from YAML.")
 @click.argument("stack_name", type=str, required=True)
 @click.argument("filename", type=str, required=False)
+@click.option(
+    "--ignore-version-mismatch",
+    is_flag=True,
+    help="Import stack components even if the installed version of ZenML "
+    "is different from the one specified in the stack YAML file",
+)
 @click.pass_context
 def import_stack(
-    ctx: click.Context, stack_name: str, filename: Optional[str]
+    ctx: click.Context,
+    stack_name: str,
+    filename: Optional[str],
+    ignore_version_mismatch: bool,
```

Comment:
One more thing id like here is this --decouple-stores flag that is bothering me a lot and is available in the zenml stack register command natively

Reply:
FYI I think this flag will be gone with the new ZenServer/MLMD changes, probably still makes sense to include/forward it here for now right?

Reply:
Okay, I have added the `decouple_stores` option.